tools/dev/server.js

```diff
@@ -14,28 +14,58 @@
 
 const webpack = require('webpack');
 const WebpackDevServer = require('webpack-dev-server');
+const selfsigned = require('selfsigned');
 const config = require('../../webpack.config.js');
 const devServers = {};
 
+const {
```

Comment:
we are going to have to change this. We still have the chat widget supporting Node 4. When we go to widget 1.7 we should include bumping the node version up as part of that.